### DISCUSSION TITLE
Akamai blocking my IP

### DISCUSSION THREAD

**Discussion by morten-holm** [teslamate-org/teslamate#2549] [General]
I'm running teslamate off of my Raspberry pi and now for the 2nd time this year Akamai has blocked my ip so that i'm not able to access auth.tesla.com.

Have any of you experienced this too?

**Comment by ktsenvolt** [comment]
I had 'Advanced installation with Traefik, Let's Encrypt & HTTP Basic Auth'installed on Raspberry  Pi and I have been blocked by certain public WiFis i.e. 'Walmart and CIBC'.
